radar/radar.go

```diff
@@ -108,47 +125,48 @@ func (radar *Radar) Scan(logger lager.Logger, resourceName string) error {
 	return radar.scan(logger, resourceName)
 }
 
-func (radar *Radar) scan(logger lager.Logger, resourceName string) error {
+func (radar *Radar) scan(logger lager.Logger, resourceName string) (err error) {
 	pipelinePaused, err := radar.db.IsPaused()
 	if err != nil {
 		logger.Error("failed-to-check-if-pipeline-paused", err)
-		return err
+		return
 	}
 
 	if pipelinePaused {
 		logger.Debug("pipeline-paused")
-		return nil
+		return
 	}
 
-	config, _, err := radar.db.GetConfig()
+	savedResource, err := radar.db.GetResource(resourceName)
 	if err != nil {
-		logger.Error("failed-to-get-config", err)
-		// don't propagate error; we can just retry next tick
-		return nil
+		return
```

Comment:
bug here introduced by named returns: err return value should be nil. or did you mean for this change?